bitnami/grafana-operator/templates/deployment.yaml

```
@@ -134,12 +125,10 @@ spec:
             - name: TEMPLATE_PATH
               value: /usr/local/bin/templates
             - name: WATCH_NAMESPACE
-              {{- if .Values.operator.watchNamespace }}
-              value: {{ .Values.operator.watchNamespace }}
-              {{- else }}
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
+              {{- if and .Values.namespaceScope (eq .Values.watchNamespaces "") }}
```

IIRC, this should be the same and is the go-to syntax we use in the repo.

```suggestion
              {{- if and .Values.namespaceScope (empty .Values.watchNamespaces) }}
```